docs/changelog-fragments.d/454.misc.md

```diff
@@ -0,0 +1,4 @@
+Added changelog fragment management infrastructure using [Towncrier]
+-- by {user}`webknjaz`
```

Comment:
The user role is set up on the Sphinx side and it links the person's GH profile or Sponsors page (if exists).